.github/workflows/release.yml

```diff
@@ -41,7 +41,6 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           pullRequests: "false"
-          onlyLastTag: "true"
```

Comment:
Did you test this? I believe that if you remove this it will generate the changelog with all the history, while we only need the log from the last tag (which is the one being released)

Reply:
OK, maybe I misunderstood the option here.